Sources/Classes/core/MiniAppAnalytics.swift

```diff
@@ -12,6 +12,8 @@ internal enum MiniAppRATEvent: String, CaseIterable {
     case close
     case signatureFailure = "mini_app_signature_validation_fail"
     case getUniqueId = "mini_app_get_unique_id"
+    case getContactId = "mini_app_get_contact_id"
+    case getMauid = "mini_app_get_mauid_id"
```

Comment:
can we rename `mini_app_get_mauid_id` to `mini_app_get_mauid`? I use `mini_app_get_mauid` in android.